esmf_regrid/experimental/unstructured_scheme.py

```diff
@@ -101,8 +97,34 @@ def _create_cube(data, src_cube, mesh_dim, grid_x, grid_y):
 
     new_cube.metadata = copy.deepcopy(src_cube.metadata)
 
-    for coord in src_cube.coords(dimensions=()):
-        new_cube.add_aux_coord(coord.copy())
+    # TODO: Handle derived coordinates. The following code is taken from
+    #  iris, the parts dealing with derived coordinates have been
+    #  commented out for the time being.
+    # coord_mapping = {}
+
+    def copy_coords(src_coords, add_method):
+        for coord in src_coords:
+            dims = src_cube.coord_dims(coord)
+            if hasattr(coord, "mesh") or mesh_dim in dims:
+                continue
+            dims = [dim if dim < mesh_dim else dim + 1 for dim in dims]
```

Comment:
Can we have a comment to explain this line?  
I think from the `continue` statement above, `dim` will never equal `mesh_dim`, but explaining why it's going to be necessary to shift all later dimensions by 1 would make it clearer.

Reply:
If this code has been copied from iris then this comment probably belongs there too!

Reply:
This line is new.